modules/tinymce/src/plugins/fullscreen/main/ts/core/Actions.ts

```diff
@@ -132,7 +132,10 @@ const toggleFullscreen = (editor: Editor, fullscreenState: Cell<ScrollInfo | nul
     handleClasses(DOM.removeClass);
 
     viewportUpdate.unbind();
-    Optional.from(fullscreenState.get()).each((info) => info.fullscreenChangeHandler.unbind());
+    const info = Maybes.from(fullscreenState.get());
+    if (Maybes.isJust(info)) {
```

Comment:
Hm. I can't remember from the discussions - did we implement an `each` equivalent for Maybes? Might be that I'm just used to the old style, but switching from `each` to an if here feels really odd to me.